tornado-drivers/ptx/src/main/java/uk/ac/manchester/tornado/drivers/ptx/PTXDeviceContext.java

```diff
@@ -152,6 +152,11 @@ public int getDriverIndex() {
         return TornadoRuntime.getTornadoRuntime().getBackendIndex(PTXBackendImpl.class);
     }
 
+    @Override
+    public Set<Long> getRegisteredPlanIds() {
+        return Set.of();
```

Comment:
Why does this class do not store the `executionIDs` similar to the other two (OCLDeviceContext, SPIRVDeviceContext)?